core/templates/dev/head/editor/exploration_settings.html

```diff
@@ -431,7 +431,8 @@
 
     <h3>
       Initial parameter changes
-      <span class="glyphicon glyphicon-info-sign" tooltip="These changes are applied before the learner begins the exploration." tooltip-placement="right" style="font-size: 12px; padding-left: 18px;"></span>
+      <i class="material-icons md-18" tooltip="These changes are applied before the learner begins the exploration."
+         tooltip-placement="right" style="padding-left: 18px;">&#xE88E;</i>
```

Comment:
I don't think you need so much padding-left here. Also, the icon does not seem correctly vertically aligned. Did you check manually?